features/buyer/catalogue.feature

```diff
@@ -73,3 +73,12 @@ Scenario: User is able to search by keywords field on the search results page to
   Then I see that service.id in the search summary text
   And I see that service.id as the value of the 'q' field
   And I see that service in the search results
+
+Scenario: User is able to click a filter on the search results page to narrow down the results returned
+  Given I am on the /g-cloud page
+  And I have a random g-cloud service from the API
```

Comment:
Yep, so I'm very happy with the approach in your other PR. I still think we might at some point need to be able to distinguish between 'indexed' frameworks, and 'live but no longer actually in the index' frameworks. But I agree we wouldn't want to introduce something just for the sake of the functional tests.